docs: Updated environment variable docs (#63)
invertase/globe: docs/deployments/environment-variables.mdx

It is also possible to specify what environments the variable should be available in.

<Warning>
After creating or updating an environment variable, you will need to redeploy your project for the changes to take effect.
</Warning>

## Accessing environment variables

Within your Dart code, you can access environment variables using the `Platform.environment` map. Any package that uses this map can access Globe environment variables.

```dart
String? value = Platform.environment['MY_VARIABLE'];
```

### Using "dotenv" package

> If you prefer to use a `.env` file for local development, you can use the [dotenv package](https://pub.dev/packages/dotenv) to load environment variables from a file locally and use Globe environment variables when deployed.

Reviewer: ```suggestion
If you prefer to use a `.env` file for local development, you can use the [dotenv package](https://pub.dev/packages/dotenv) to load environment variables from the `.env` file locally and use Globe environment variables when deployed.
```

Author: Updated

> Add the `includePlatformEnvironment: true` to the `DotEnv` constructor to include platform environment variables when `.env` file is not defined.

Reviewer: ```suggestion
When your app is deployed to Globe, any static assets are ignored, so you need to pass `includePlatformEnvironment: true` to the `DotEnv` constructor to include platform environment variables that you previously defined on Globe dashboard (Project > Settings > Environment variables).
```

Author: Updated

```dart
final env = DotEnv(includePlatformEnvironment: true)..load();
String? value = env['MY_VARIABLE'];
```

> ### Using "envied" package
> 
> Currently the Globe environment variables does not support the [envied package](https://pub.dev/packages/envied). This is because the Globe environment variables are not available during the build process, and `envied_generator` requires it. We are currently working on adding support for it.
> 
> Check [this issue](https://github.com/invertase/globe/issues/59) for more information. Upvote if this is something you would like to see.

Reviewer: ```suggestion
```

Reviewer: I'll be fixing this very soon, so not worth adding this not just to remove it.

Author: Removed

## System environment variables

> The following environment variables are available on all deployments:

Reviewer: Your deployed Dart application has access to the Globe-specific environment variables which wouldn't be typically available while you develop/test your app locally.

Author: Updated
